docs/Manifest.toml

```diff
@@ -164,9 +164,9 @@ version = "0.8.0"
 
 [[deps.ClimaComms]]
 deps = ["CUDA", "StaticArrays"]
-git-tree-sha1 = "93c4c545024821c9cacb7417f00381b2329e76db"
+git-tree-sha1 = "5ab2642608e411eaf96d5d59c49ae4ab3bcdf2df"
 uuid = "3a4d1b5c-c61d-41fd-a00a-5873ba7a1b0d"
-version = "0.3.2"
+version = "0.3.4"
 
 [[deps.ClimaCore]]
```

Comment:
Should we wait for the new ClimaCore release?

Reply:
We may have to wait. The PR has several failures that need to be resolved.